.github/PULL_REQUEST_TEMPLATE/development.md

```diff
@@ -0,0 +1,33 @@
+## Pre-Review Checklist for the PR Author
+
+1. [ ] Branch follows the naming format (`iox-#123-this-is-a-branch`)
+1. [ ] Commits messages are according to this [guideline][commit-guidelines]
+    - [ ] Commit messages have the issue ID (`iox-#123 commit text`)
+    - [ ] Commit messages are signed (`git commit -s`)
+    - [ ] Commit author matches [Eclipse Contributor Agreement][eca] (and ECA is signed)
+2. [ ] Update the PR title
+   - Follow the same conventions as for commit messages
+   - Link to the relevant issue
+3. [ ] Add sensible notes for the reviewer
+4. [ ] Assign PR to reviewer
+
+[commit-guidelines]: https://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html
+[eca]: http://www.eclipse.org/legal/ECA.php
+
+## Notes for Reviewer
+<!-- Items in addition to the checklist below that the reviewer should look for -->
+
+## Checklist for the PR Reviewer
```

Comment:
We always need at least two reviewers and sometimes even more when its a hot topic. Therefore, I would adapt this a little bit. What about adding the Checklist for the PR Reviewer multiple times so that the reviewer can enter its name (in ENTER NAME) and then work on the list. Something like:

## Checklist for the PR Reviewer - Kermit
- [ ] Commits are properly organized and messages are according to the guideline
- [ ] PR title describes the changes
- [ ] Approve

## Checklist for the PR Reviewer - Lemonhead
- [ ] Commits are properly organized and messages are according to the guideline
- [ ] PR title describes the changes
- [ ] Approve

## Checklist for the PR Reviewer - ENTER NAME
- [ ] Commits are properly organized and messages are according to the guideline
- [ ] PR title describes the changes
- [ ] Approve

Reply:
I'm not in favor of this. Do we really need to have multiple reviewers to check things like the correct PR title?

I also think we do not need the Approve checkbox and then there is nothing which is related to a single reviewer. Maybe we can change the Approve checkbox to a note like `PR will only be approved after all the checkboxes from above are ticked`

Reply:
I would propose that when one reviewer checked something and ticked the box, others do not have to do this explicitly again. The additional reviewers are more useful for the non-formal checks